react/features/base/label/components/ExpandedLabel.native.js

```diff
@@ -0,0 +1,135 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Animated, Text, View } from 'react-native';
+
+import styles, { DEFAULT_COLOR, LABEL_MARGIN, LABEL_SIZE } from './styles';
+
+export type Props = {
+
+    /**
+     * The position of the parent element (from right to left) to display the
+     * arrow.
+     */
+    parentPosition: number
+};
+
+type State = {
+
+    /**
+     * The opacity animation Object.
+     */
+    animOpacity: Object,
```

Comment:
in the previous object the animation is called `pulseAnimation`, here we put `anim` first, which is abreviated... please pick one, but let's make sure it's consistent.